tests/end-to-end/regression/GitHub/3396/issue-3396-test.phpt

```diff
@@ -19,34 +19,34 @@ PHPUnit\TextUI\Command::main();
-Test 'DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' started
-Test 'DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' ended
-Test 'DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' started
-Test 'DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' ended
-Test 'DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' started
-Test 'DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' ended
-Test 'DataproviderExecutionOrderTest::testFirstTestThatAlwaysWorks' started
-Test 'DataproviderExecutionOrderTest::testFirstTestThatAlwaysWorks' ended
-Test 'DataproviderExecutionOrderTest::testTestInTheMiddleThatAlwaysWorks' started
-Test 'DataproviderExecutionOrderTest::testTestInTheMiddleThatAlwaysWorks' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testFirstTestThatAlwaysWorks' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testFirstTestThatAlwaysWorks' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testTestInTheMiddleThatAlwaysWorks' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testTestInTheMiddleThatAlwaysWorks' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' ended
```

Comment:
@epdenouden  @sebastianbergmann 

I have no idea why the order would be different when `DataproviderExecutionOrderTest` is moved into the `PHPUnit\TestFixture` namespace. Do you?

Reply:
No idea. AFAIK, the test no longer exists in `master` so I would not worry about it too much.